packages/instant/src/components/instant_heading.tsx

```diff
@@ -113,20 +113,23 @@ export class InstantHeading extends React.Component<InstantHeadingProps, {}> {
     }
 
     private readonly _renderEthAmount = (): React.ReactNode => {
+        const ethAmount = format.ethBaseUnitAmount(
+            this.props.totalEthBaseUnitAmount,
+            4,
+            <AmountPlaceholder isPulsating={false} color={PLACEHOLDER_COLOR} />,
+        );
+
+        const fontSize = typeof ethAmount === 'string' && ethAmount.length >= 13 ? '14px' : '16px';
```

Comment:
I prefer to use the vanilla JS here as its only a little more verbose, esp. after our eng discussion about potentially removing lodash from packages in the future

Reply:
If we removed lodash, we would probably use something else for this check. Also checking for string type in javascript is a bit tricky. https://stackoverflow.com/questions/4059147/check-if-a-variable-is-a-string-in-javascript 

We would probably never run into the edge cases, but since we are already using `_.isString` in several places, it is technically more correct, and is more readable, I would prefer to use `_.isString`. Also, if and when we move to Ramda, we want to "tag" this instance for easy changing.